result/bmpprefix.yaml

```diff
@@ -0,0 +1,652 @@
+          items:
+            type: string
+            x-enum:
+              pre_policy:
+                x-field-uid: 1
+              post_policy:
+                x-field-uid: 2
+          x-field-uid: 5
+    BmpPrefix.Ipv4Unicast.Filter:
+      type: object
+      properties:
+        addresses:
+          description: >-
+            The addresses to match. If the addresses property is missing or
+            empty then all addresses will match.
+          type: array
+          items:
+            type: string
+            format: ipv4
+          x-field-uid: 1
+        prefix_length:
+          x-include: '#/components/schemas/BmpPrefix.Criteria.Base/properties/prefix_length'
+          x-field-uid: 2
+        origin:
+          x-include: '#/components/schemas/BmpPrefix.Criteria.Base/properties/origin'
+          x-field-uid: 3
+        path_id:
+          x-include: '#/components/schemas/BmpPrefix.Criteria.Base/properties/path_id'
+          x-field-uid: 4
+    BmpPrefix.Ipv6Unicast.Filter:
+      type: object
+      properties:
+        addresses:
+          description: >-
+            The addresses to match. If the addresses property is missing or
+            empty then all addresses will match.
+          type: array
+          items:
+            type: string
+            format: ipv6
+          x-field-uid: 1
+        prefix_length:
+          x-include: '#/components/schemas/BmpPrefix.Criteria.Base/properties/prefix_length'
+          x-field-uid: 2
+        origin:
+          x-include: '#/components/schemas/BmpPrefix.Criteria.Base/properties/origin'
+          x-field-uid: 3
+        path_id:
+          x-include: '#/components/schemas/BmpPrefix.Criteria.Base/properties/path_id'
+          x-field-uid: 4
+    BmpPrefix.Criteria.Base:
+      type: object
+      properties:
+        prefix_length:
+          description: >-
+            The prefix length to match. If the prefix length is missing then all
+            prefix lengths will match.
+          type: integer
+          format: uint32
+          maximum: 128
+          x-field-uid: 1
+        origin:
+          description: >-
+            The origin to match. If the origin is missing then all
+            origins will match.
+          type: string
+          x-field-uid: 2
+          x-enum:
+            igp:
+              x-field-uid: 1
+            egp:
+              x-field-uid: 2
+            incomplete:
+              x-field-uid: 3
+        path_id:
+          description: >-
+            The path id to match. If the path id is missing then all
+            path ids will match.
+          type: integer
+          format: uint32
+          x-field-uid: 3
+    BmpPrefixes.State:
+      description: >-
+        BMP server learned bgp peer prefixes.
+      type: object
+      properties:
+        peer_bgp_id:
+          description: >-
+            The Identifier of a BGP peer.
+          type: string
+          x-field-uid: 1
+        ipv4_unicast_prefixes:
```

Comment:
I think a pre-policy and post-policy view of routes would be more useful [ within BMP server --> per server per peer as mentioned ]